src/octoprint/timelapse.py

```diff
@@ -838,7 +839,40 @@ def _render(self):
 		with self.render_job_lock:
 			try:
 				self._notify_callback("start", output)
-				p = sarge.run(command_str, stdout=sarge.Capture(), stderr=sarge.Capture())
+				p = sarge.run(command_str, stdout=sarge.Capture(), stderr=sarge.Capture(), async=True)
+
+				self._logger.debug("Parsing ffmpeg output")
+
+				# Compile regex to make it faster
+				duration_regex = re.compile("Duration: (\d{2}):(\d{2}):(\d{2})\.\d{2}")
+				current_regex = re.compile("time=(\d{2}):(\d{2}):(\d{2})\.\d{2}")
+				duration_s = ""
```

Comment:
Shouldn't this rather be initialized to `0`? An empty string definitely feels very wrong here, especially since further down we try to parse that into a `float` which would throw a `ValueError`.